src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -310,16 +310,16 @@ export default {
 		 * Calculates the total width of all breadcrumbs
 		 *
 		 * @param {Array} breadcrumbs All breadcrumbs
-		 * @returns {Integer} The total width
+		 * @return {bigint} The total width
```

Comment:
Shouldn't it be `number`?

Same in other places.

Reply:
Yes, indeed here it's a number. In some other places it really is an integer. But I am fine with writing number everywhere.

Reply:
I changed it to number everywhere.